app/src/main/res/layout-sw600dp-port/promoted_story_list.xml

```diff
@@ -61,7 +61,7 @@
       android:scrollbars="none"
       android:orientation="horizontal"
       android:paddingStart="@dimen/home_padding_start"
-      android:paddingEnd="@{viewModel.endPadding}"
```

Comment:
If this worked before, do we actually need the new binding adapter?

Reply:
Done